### ISSUE TITLE
Web page for pint default units

### ISSUE THREAD

**Issue by nh916** [hgrecco/pint#1571]
Is there a chance we can have a web page for the supported units for pint for our documentation? 

The txt file is awesome, but I sometimes feel intimidated looking at it. I know if I am working with clients and want to show them all the supported units they can use with the my program, I'd have to point them to the pint txt file which they'll feel overwhelmed very quickly.

This is not high priority or anything, and is more of a request than an issue, but if we can have a friendly and easy list for the documentation that would be super helpful!

**Comment by OrangeChannel** [comment]
I don't think this makes much sense and would be a pain to manage and document. Every time the default units get updated we would need to update the txt file (or hopefully toml in the near future) *and* the doc files, and how would the documentation look anyways? We shouldn't be responsible for describing units or systems, how they are used, what they are used for etc. 

It would be nice to have a definitive source for all units, systems, dimensions, things like that but I think it's way out of scope currently. Afaik, current devs don't really like adding somewhat obscure units/constants to the library anyways, so not even we would cover everything if this was done.

This may all change if we decide to go down that route at some point, though.
